# this has joint poses and velocities, and swimmer pos and heading

float32 joint1pos
float32 joint1vel
float32 joint2pos
float32 joint2vel
float32 swimmer_x
float32 swimmer_y
float32 swimmer_direction
float32 swimmervel_x
float32 swimmervel_y
float32 swimmervel_ang
uint8 hedgehog_count